### ISSUE TITLE
Request specific MUD file

### ISSUE THREAD

**Issue by anunezze** [CIRALabs/Home-Network-Registry-Gateway#11]
As a MUD supervisor I want to request an specific MUD file so that I can have knowledge from the vendor about the device operations.

Requirements:

- [ ] Add requirement here

**Comment by mcr** [comment]
I'm rather vague about this requirement.   How would one do this?  The current model is that either the MUD URL arrives via DHCP/BRSKI/LLDP, or it is input on the mobile app via QR code. Of course, a sophisticated user could paste the URL directly.  (Perhaps we should consider if we should have an Intent (Android thing) so that MUD URLs could be provide in email, and, when clicked, open the SHG App, rather than a browser. We could also register a content-type handler)

**Comment by anunezze** [comment]
Yes this story is after the MUD supervisor software gets the MUD URL.
